code/modules/admin/admin_attack_log.dm

```diff
@@ -3,7 +3,7 @@
 /mob/var/attack_log = list( )
 /mob/var/dialogue_log = list( )
 
-/proc/log_and_message_admins(var/message as text, var/mob/user = usr)
+/proc/log_and_message_admins(var/message as text, var/mob/user)
```

Comment:
Offhand I think this will break a bunch of existing uses where usr is expected to be available from being called in a verb or proc as verb's tree.